requirements-dev.txt

```diff
@@ -11,3 +11,5 @@ click==6.7
 twine==1.11.0
 wheel==0.33.6
 prompt_toolkit==3.0.5
+sshtunnel==0.4.0
+psycopg==3.0.11
```

Comment:
Does psycopg2 needs to be pinned to an exact version?

Reply:
I don't think it does. I can change it to point to >= 3

Reply:
>= 3.0.11 is better, as 3.0.10 and 3.0.11 have released fixes for bugs raised by pgcli.

Reply:
Wait, this shouldn't be needed at all, because the [porting PR](https://github.com/dbcli/pgcli/pull/1324) is still unmerged. If we have `psycopg` referenced in tests somewhere, this is an error. Main still has `psycopg2`.

Reply:
Sorry for the confusion. I'll remove it. It was because of `pgspecial`. It wouldn't install `pgcli` as editable because of `pscopg2` so I had to manually install it `pgspecial` which brought the dependency for `psycopg`

Reply:
Yes, please remove both `psycopg` and `sshtunnel` from `requirements-dev.txt`. When you install `pgcli`, it installs `psycopg2`. As for `sshtunnel`, instead of adding it to dev reqs, we need to update the contributing instructions to tell developers to install things the same way it's done in CI:

https://github.com/dbcli/pgcli/blob/366c01bbb97fee96a8169629854af7c83d626c99/.github/workflows/ci.yml#L40-L43

Also, the version of `pgspecial` that's currently in `main` branch is already ported to `psycopg`. So you can't use it with `main` branch of pgcli. You need the last version of `pgspecial that still uses `psycopg2`, that's 1.13.1.

Reply:
I've removed the changes form the `requirements-dev.txt`. But I did have to add `sshtunnel` as a `deps` in `tox.ini` for Tox to run.